src/components/Comments/Comments.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment/Comment';
+
+import './Comments.css';
+
+export const Comments = ({ comments }) => (
+  <ul className="comments">
+    {comments.map(comment => <Comment key={comment.id} {...comment} />)}
```

Comment:
давай краще записувати так
```suggestion
    {comments.map(comment => <Comment key={comment.id} comment={comment} />)}
```
Так зрозуміліше, що передається